packages/TelegramClient-Core.package/TCCChat.class/instance/isNotPrivate.st

```diff
@@ -1,4 +0,0 @@
-initialization
-isNotPrivate
-
-	^ self isPrivate not
```

Comment:
Why did you delete this?

Reply:
We added the `isGroup` so this was not used anymore. I guess we can add the negation again later if we need it. On the other hand, `isGroup` is already the counter part of this, but that might change in the future.